### PR TITLE
Readme.md file for the m-cluster

### PR DESCRIPTION
**Pull request by wajihahmed**

### CODE REVIEW

File: 6.0/m-cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+### AM Config
+- Single realm
+- OAuth 2.0 Server and Client
+- Separate userstore, configstore and ctsstore
+- LDAP Auth module default
+
+### IDM Config
+- Postgres as repo
+- Syncronization to LDAP
```
Comment (dgoldssfo) [inline 6.0/m-cluster/README.md]: s/Syncronization/Synchronization/ (typo)

File: 6.5/m-cluster/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+### AM Config
+- Single realm
+- OAuth 2.0 Server and Client
+- Separate userstore, configstore and ctsstore
+- LDAP Auth module default
+
+### IDM Config
+- Postgres as repo
+- Syncronization to LDAP
```
Comment (dgoldssfo) [inline 6.5/m-cluster/README.md]: Same typo as above

File: 6.0/m-cluster/README.md (hunk 1 above)
Comment (wajihahmed) [reply 6.0/m-cluster/README.md]: will fix in the next PR

File: 6.5/m-cluster/README.md (hunk 2 above)
Comment (wajihahmed) [reply 6.5/m-cluster/README.md]: will fix in the next PR

**Pull request merged**